### PR TITLE
[IMP] l10n_in_edi_ewaybill: on verify it's only display error code

### PR DESCRIPTION
**Pull request by zepa-odoo**
Before This PR:
--------------------
If the username and password are wrong, then it only displays the error "108".

After This PR:
------------------
If the username and password are wrong, then it displays the error.
"[108] Invalid login credentials."

task - 3304862

### CODE REVIEW

**Review by jva-odoo — APPROVED** [review]

File: addons/l10n_in_edi_ewaybill/models/account_edi_format.py [hunk 1]
```diff
@@ -540,6 +540,12 @@ def _l10n_in_edi_ewaybill_check_authentication(self, company):
                 return True
         return False
 
+    def _get_error_message(self, response):
```
Comment (jco-odoo) [inline addons/l10n_in_edi_ewaybill/models/account_edi_format.py:543]:
```suggestion
    def _l10n_in_get_error_message(self, response):
``` 
@zepa-odoo

Comment (zepa-odoo) [reply addons/l10n_in_edi_ewaybill/models/account_edi_format.py]: @jco-odoo Changes Done.

File: addons/l10n_in_edi_ewaybill/models/account_edi_format.py [hunk 2]
```diff
@@ -540,6 +540,12 @@ def _l10n_in_edi_ewaybill_check_authentication(self, company):
                 return True
         return False
 
+    def _l10n_in_get_error_message(self, response):
```
Comment (jco-odoo) [inline addons/l10n_in_edi_ewaybill/models/account_edi_format.py:543]: @zepa-odoo Looking at it more closely, you are actually setting the error message in the response
```suggestion
    def _l10n_in_set_missing_error_message(self, response):
```

Comment (zepa-odoo) [reply addons/l10n_in_edi_ewaybill/models/account_edi_format.py]: Done.

**Comment by jco-odoo** [comment]
@robodoo r+

**Pull request closed without merging**